testing/scenario_app/lib/src/solid_blue.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+import 'scenario.dart';
+
+/// Fills the screen with a solid blue color.
+class SolidBlueScenario extends Scenario {
+  /// Creates the SolidBlue scenario.
+  SolidBlueScenario(super.view);
+
+  @override
+  void onBeginFrame(Duration duration) {
+    final SceneBuilder builder = SceneBuilder();
+    final PictureRecorder recorder = PictureRecorder();
+    final Canvas canvas = Canvas(recorder);
+
+    canvas.drawPaint(Paint()..color = const Color(0xFF0000FF));
```

Comment:
This is in BGRA?  I would have expected RGBA at this level, probably worth commenting.

Reply:
It's ARGB - this is the normal dart:ui color class...